geolocator/example/lib/main.dart

```diff
@@ -201,10 +252,7 @@ class _GeolocatorWidgetState extends State<GeolocatorWidget> {
   }
 }
 
-enum _PositionItemType {
-  permission,
-  position,
-}
+enum _PositionItemType { permission, position, locationservicestatus }
```

Comment:
Dart style guide suggests to use lower camel casing for enum values. Meaning the initial letter of a word is written in lower case, and initial letter of sub-sequent words are written with in upper case. Also add a comma after the last enum so formatting will put each element on its own line:
```suggestion
enum _PositionItemType { 
  permission,
  position,
  locationServiceStatus, 
}
```